lib/src/facts/linux/networking_resolver.cc

```diff
@@ -143,6 +143,13 @@ namespace facter { namespace facts { namespace linux {
             vector<boost::iterator_range<string::iterator>> parts;
             boost::split(parts, line, boost::is_space(), boost::token_compress_on);
```

Comment:
This is where we're ignoring the extra space in the middle of the line. You can change this to `boost::token_compress_off`